templates/contribute/index.md

```diff
@@ -23,6 +27,13 @@ to make the process of contributing as smooth as possible.
 
 - See [Caching compilation](#caching-compilation) for commands to automatically call `leanproject get-cache`.
 
+## Working with an existing PR
+We use GitHub "labels" to manage review. On the main page for a PR, on the right-hand side, 
+there should be a sidebar with panels "reviewers", "asignees", "labels", etc. 
+Click on the "labels" header to add or remove labels from the current project. The most important labels are "request-review" and "changes-requested". If you label your PR with "request-review", someone will probably "review" it within a few days (depending on the size of the PR; smaller PRs will get quicker responses). The reviewer will probably leave comments and change the label to "changes-requested". You should address each comment, clicking the "resolve conversation" button once the problem is resolved. Ideally each problem is resolved with a new commit, but there is no hard rule here. Once all requested changes are implemented, you should change the label back to "request-review" to start the process over again.
+
+After some iteration, a reviewer will "approve" the PR. A bot called `bors` will make a comment with instructions. After following these, the PR will get added to the "merge queue". The merge queue gets cleared automatically by bors, but this takes some finite amount of time as it requires building branches of mathlib.
```

Comment:
We have a little intro to `bors` [here](https://github.com/leanprover-community/mathlib/blob/master/docs/contribute/bors.md). It might be too much for new contributors though.

Reply:
We can still have a link for people who want details.